release-notes/release-3.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: February 28, 2024
+---
+
+# Features
```

Comment:
Other noteworhty ites:
1. Mobile RUM Session Replay, that's huge
2. log filter values will update based on other filter values selected (not available when selecting log level as a filter)